libraries/lib-components/EffectInterface.h

```diff
@@ -391,7 +391,7 @@ class COMPONENTS_API EffectInstance
     Other member functions related to realtime return true or zero, but will not
     be called, unless a derived class overrides RealtimeInitialize.
     */
-   virtual bool RealtimeInitialize(EffectSettings &settings);
+   virtual bool RealtimeInitialize(EffectSettings &settings, double sampleRate);
```

Comment:
I always wondered why we use double for sample rate. I've never seen non integer rates

Reply:
The AudioUnit library uses double, who knows why.